packages/app/src/components/DimensionsPanel/Dialogs/DialogManager.js

```diff
@@ -81,23 +165,53 @@ export class DialogManager extends Component {
     };
 
     renderDialogContent = () => {
-        const { dialogId } = this.props;
+        const {
+            displayNameProperty,
+            dialogId,
+            dimensions,
+            removeUiItems,
+            setUiItems,
+        } = this.props;
+
+        const dimensionProps = {
+            d2: this.context.d2,
+            onSelect: this.onSelect,
+            onDeselect: removeUiItems,
+            onReorder: setUiItems,
+        };
+
         const dynamicContent = () => {
+            const selectedItems = this.getSelectedItems(dialogId);
+
             if (dialogId === dxId) {
-                return <DataDimension />;
+                return (
+                    <DataDimension
+                        displayNameProp={displayNameProperty} // XXX from user settings but metadata only has name
```

Comment:
Is this comment still relevant?

Reply:
No, if `displayName` is available it's always added to the object stored in metadata.